core/input/input.cpp

```diff
@@ -718,6 +718,7 @@ void Input::_parse_input_event_impl(const Ref<InputEvent> &p_event, bool p_is_em
 		}
 		ERR_FAIL_COND_MSG(event_index >= (int)MAX_EVENT, vformat("Input singleton does not support more than %d events assigned to an action.", MAX_EVENT));
 
+		print_line(p_event);
```

Comment:
It does not make any sense, but this line is required for some reason:

https://github.com/godotengine/godot/assets/2223172/c7f0b130-b95e-4892-9692-ecc8adcf9d69

I tested it multiple times, always getting the same result. This is the MRP from the issue.

Reply:
I can confirm, that for me on Linux, with the print_line the sprite moves and without the print_line the sprite doesn't move.

Reply:
I think its because in `InputMap::event_get_action_status` pressed_state isn't set, so `InputEvent::is_action_pressed` uses unset garbage memory, which was affected by the print line.

Reply:
Ok fixed. Apparently InputEventAction was already handled below my code, so I just integrated the event index into it.